controllers/complianceeventsapi/server.go

```diff
@@ -526,6 +673,21 @@ func getSingleComplianceEvent(db *sql.DB, w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	// Check auth for managedCluster GET verb
+	isAuth, err := getSingleManagedClusterRules(config, complianceEvent.Cluster.Name)
+	if err != nil {
+		log.Error(err, "Failed to get the Read authorization for "+complianceEvent.Cluster.Name)
+		writeErrMsgJSON(w, "Cannot get read Authorization", http.StatusInternalServerError)
```

Comment:
We don't want to expose this to the user.
```suggestion
		writeErrMsgJSON(w, "Internal Error", http.StatusInternalServerError)
```